client/view/Form/form.js

```diff
@@ -17,8 +18,8 @@ const errorMessage = document.querySelector('[data-id=error_message]')
 
 form.addEventListener('submit', handleFormSubmit)
 
-textarea.addEventListener('keypress', e => {
-  if (e.key === 'Enter' && !e.shiftKey) {
+textarea.addEventListener('keydown', e => {
```

Comment:
Mobile phones has no `Shift + Enter` so we will need to auto-send any valid query to the server